service/telemetry/config.go

```diff
@@ -94,6 +94,23 @@ type LogsConfig struct {
 	//
 	// By default, there is no initial field.
 	InitialFields map[string]any `mapstructure:"initial_fields"`
+
+	// Filtering rules to exclude log statements. See https://github.com/moul/zapfilter
+	// for documentation.
+	//
+	// Example:
+	//
+	//      filter_rules: "*:myns info,warn:myns.* error:*"
+	// By default, there is no filter.
+	FilterRules string `mapstructure:"filter_rules"`
+
+	// Filtering regular expression to exclude log statements.
+	//
+	// Example:
+	//
+	//      filter_regex: "partition.*"
+	// By default, there is no regular expression filter.
+	FilterRegex string `mapstructure:"filter_regex"`
```

Comment:
Do we need regexp? Can we keep the blob only for now? If we will need regexp, we can apply the same approach as in the signalfx exporter and read values like `/regexp/` from the same `exclude` field.

Reply:
We don't have a blob approach right now, did I miss something?